orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/tasks/StartPipelineTask.groovy

```diff
@@ -50,7 +50,8 @@ class StartPipelineTask implements Task {
     Map<String, Object> pipelineConfig = pipelines.find { it.id == pipelineId }
 
     if (!pipelineConfig) {
-      throw new IllegalArgumentException("Unable to locate referenced pipeline $pipelineId.")
+      throw new IllegalArgumentException("${isStrategy ? 'Custom strategy selected but ' : ''}" +
```

Comment:
Will this not render:

`Custom strategy selected but Unable to locate referenced pipeline $pipelineId.`

That still doesn't seem quite right.

`The referenced ${isStrategy ? 'custom strategy' : 'pipeline'} cannot be located (${pipelineId})`

Reply:
Much clearer, thanks. Updated.